p4-16/spec/P4-16-spec.mdk

```diff
@@ -4054,7 +4054,7 @@ the `struct` or `header` types do need to be initialized though.
 
 Some P4 expressions denote sets of values (`set<T>`, for some type `T`;
 see Section [#sec-set-types]). These expressions can
-appear only in a few contexts---parsers and constant table
```

Comment:
This and the next small change in this PR are not strictly necessary, but the table property name is technically `entries`, not `const entries`, although currently the `const` is required whenever `entries` appears.  You can also consider these minor changes as prelude to defining new semantics for `entries` without `const`, which I hope to present to the LDWG next Monday (first Monday of October), if there is time available in the meeting's agenda.